### ISSUE TITLE
cannot open xml file on my server using url because of proxy

### ISSUE THREAD

**Issue by mkdhabale** [jgraph/drawio#566]
I have the same issue mentioned in the below thread,
https://github.com/jgraph/drawio/issues/255

it's give proxy error on my server but on https://draw.io/ it's working fine.

here is example url's,

    https://draw.io/?url=http://www.xyz.com/demo.xml -> Working
    https://xyz.com/?url=http://www.xyz.com/demo.xml

Note : Here xyz.com is my server and try to use xml from another server

**I am using asp.net technology in IIS server,**
Hot to set proxy in this, Is there any change in XML file for servlet.
![image](https://user-images.githubusercontent.com/11902946/65253526-61eb0100-db18-11e9-86d9-6e194cb0b886.png)

Or any other way to implement proxy in this server and this will work,

 I tried to directly pass the file using App.min.js and it gives proper response as accepted,
still it is showing file not found error,

Basically i want to open xml file to edit in this.

**Comment by mkdhabale** [comment]
Issue resolved created a http handler code by own in asp.net to handle file url data
